src/kajal/BankingSystem.java

```diff
@@ -0,0 +1,76 @@
+package kajal;
+
+public class BankingSystem {
+	String name;
+	int debitcount;
+	int creditcount;
+	int printcount = 0;
+	int totalbal;
+	static int totalcreditcount;
+	static int totaldebitcount;
+	static int totalprintcount;
+	
+		void setData(String nm, int initAmt) {
+		name= nm;
+		totalbal = initAmt;
+		}
+	
+		void debitAmt(int amt) {
+		debitcount=0;
+		
+		while(totalbal >= amt) {
```

Comment:
Why while loop here?
you should write if condition to check whether balance is sufficient or not.